.github/workflows/generate-headers.yml

```diff
@@ -9,11 +9,9 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: install nightly toolchain
-      uses: actions-rs/toolchain@v1
+      uses: dtolnay/rust-toolchain@master
       with:
```

Comment:
The following lines should then be removed.

Reply:
Done.